packages/@aws-cdk/aws-apigatewayv2/README.md

```diff
@@ -97,6 +97,14 @@ httpApi.addRoutes({
 
 The URL to the endpoint can be retrieved via the `apiEndpoint` attribute.
```

Comment:
What is the value returned by `apiEndpoint` when `disableExecuteApiEndpoint` is set?

Reply:
Good question. I deployed a test stack to check this behavior. `apiEndpoint` still returns the `execute-api` URL, so no changes there.

Reply:
And I suppose the endpoint is not reachable.

It might be a better/correct experience to throw an error when `apiEndpoint` is accessed when `disableExecuteApiEndpoint` is set.

Reply:
Good shout, I've amended the code to follow that proposal. 

This leaves me wondering what we may want to do with `fromHttpApiAttributes`. I see that it currently expects `apiEndpoint` to be provided in the attributes to import an HTTP API from another stack. Feels like we may want to modify this behavior to consider APIs without the default endpoint enabled, but I wanted to hear your thoughts before I proceed?

Reply:
From what I can tell, `apiEndpoint` is an optional attribute of `HttpApiAttributes`, so what's already there should work just as well.